core/src/main/java/org/apache/hadoop/shaded/net/jpountz/lz4/LZ4Compressor.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.shaded.net.jpountz.lz4;
```

Comment:
BTW, is there a chance of conflicts with Hadoop library when the users add it explicitly, @viirya ?

Reply:
lz4-java dependency? No, because lz4-java is relocated currently in Hadoop shaded libraries. But it is provided so it actually isn't included in the libraries. If they add lz4-java, it might conflict with Spark's, not Hadoop's.

If you mean users add this wrapper classes `org.apache.hadoop.shaded.net.jpountz.lz4.*`? I cannot image why users want to add them. If they really add, it might conflict with what we added here, yes. But as you see, the wrapper classes don't do real work but delegates to lz4-java classes, so even they are conflicted, seems to me it is not big deal as the ones added by users should also work similarly (as wrapper only).

Reply:
You are right. Thanks!